test/report/test-report-writereport.js

```diff
@@ -37,6 +37,14 @@ function validate() {
   validate();
 }
 
+{
+  // Test with an error with one line stack
+  const error = new Error();
+  error.stack = 'only one line';
+  process.report.writeReport(error);
```

Comment:
A simpler test might be to use `getReport(error)` in [`test/report/test-report-getreport.js`](https://github.com/nodejs/node/blob/master/test/report/test-report-getreport.js). Or maybe we could do both to test both `writeReport()` and `getReport()`.

Reply:
I test both now